rust_build_utils/rust_utils_config.py

```diff
@@ -70,6 +70,15 @@
         },
         "post_build": ["rust_build_utils.linux_build_utils.strip"],
     },
+    "qnap": {
+        "archs": {
+            "x86_64": {
+                "strip_path": "/usr/bin/objcopy",
+                "rust_target": "x86_64-unknown-linux-gnu",
+            },
+        },
+        "post_build": ["rust_build_utils.linux_build_utils.strip"],
+    },
```

Comment:
This looks like normal `x86_64` + `linux` defined above? Why do we need duplicated entry with a different name?

Reply:
Indeed it matches `linux x86_64` config but not in [LIBTELIO_CONFIG](https://github.com/NordSecurity/libtelio/blob/main/ci/build_libtelio.py#L144) which should have a respective `qnap` match here on `GLOBAL_CONFIG`.

Reply:
`main` branch I've reffed doesn't have `qnap` entry but will have in a related PR: https://github.com/NordSecurity/libtelio/blob/2e7ceb9d0a1593a297efa1d924c53437cd147d11/ci/build_libtelio.py#L263